src/EventSubscriber/CacheableResponseSubscriber.php

```diff
@@ -90,13 +90,22 @@ public function onRespond(ResponseEvent $event) {
       }
 
       $tags_string = implode(' ', $tags);
-      if (25000 < strlen($tags_string)) {
-        $tags_string = substr($tags_string, 0, 25000);
-        // The string might have cut of in the middle of a tag.
-        // So now find the the last occurence of a space and cut to that length.
+      $limit = 25000;
```

Comment:
25000 is appearing in a number of places now, it would be a good idea to define a constant for it.

Reply:
I guess we can't de-dup the occurrences in the yml files, though.

Reply:
Indeed, I have added a doc line to remind of the presence of this value in the config yml files

Reply:
```suggestion
      $default_limit = 25000;
```
Suggest setting $default_limit above $limit so we can use both in sprintf below.

Reply:
Thanks! I've updated the warning message to show the actual/config limit